src/models/foods.js

```diff
@@ -0,0 +1,42 @@
+'ust strict';
+
+class ThingsModel {
+    constructor() {
+this.id = 0;
+this.db = [];
+    }
+//read -> app.get -> crud -> read
+    get(id) {
+        if (id) {
+            return this.db.find(record => record.id ===id);
+        } else {
+            return this.db;
+        }
+    }
+
+//create -> app.put
+    create(obj) {
+        let record = {
+            id: ++this.id,
+            record: obj
```

Comment:
I would do a {} because you never defined the obj also it already within an object.